zilliqa/src/state.rs

```diff
@@ -176,6 +185,12 @@ impl State {
         )?;
         state.force_deploy_contract_evm(deposit_data, Some(contract_addr::DEPOSIT))?;
 
+        // Set DEPOSIT contract to total deposited at genesis
+        state.mutate_account(contract_addr::DEPOSIT, |a| {
```

Comment:
I think we should do this instead by changing the deposit contract constructor to require a `msg.value` equal to the deposits specified and then pass the value in the transaction call above.

Reply:
Alright. I'll try that in the follow up.

Reply:
Fixed in #1948